README.md

```diff
@@ -8,20 +8,42 @@ server-side symbol files (.SYM files).
 This project is intended as a drop-in replacement for
 [Snappy Symbolication Server](https://github.com/mozilla/Snappy-Symbolication-Server/)
 
-Quick Start Guide
------------------
+Quick Start without Docker
+--------------------------
 
 1. Install [Python 2.7](https://www.python.org/downloads/) if it is not already
    installed
-2. Install dependencies:
-   `python -m pip install tornado futures python-memcached psutil`
+2. Install dependencies: `python -m pip install -r requirements.txt`
```

Comment:
Freshly installed Python does come with pip: [https://pip.pypa.io/en/stable/installing/](https://pip.pypa.io/en/stable/installing/)
But it is possible that they user still might not have it installed, so I'll add it.